eelbrain/plot/_glassbrain.py

```diff
@@ -388,6 +518,10 @@ def butterfly(
             different colors. See `examples <http://nilearn.github.io/auto_examples/
             01_plotting/plot_demo_glass_brain_extensive.html>`_. Only affects
             GlassBrain plot.
+        draw_arrows: boolean
+            Draw arrows in the direction of activation over the glassbrain plots.
+            Naturally, for this to work ``ndvar`` needs to contain space dimension
+            (i.e 3D vectors). By default it is set to (``True``).
```

Comment:
Shouldn't it be this?
```suggestion
            (i.e 3D vectors). By default it is set to ``True``.
```

Reply:
Actually in plot_abs it was like (``False``), so I used the same. Changing it then. :)